terraform/projects/app-ecs-services/main.tf

```diff
@@ -75,6 +79,13 @@ data "terraform_remote_state" "app_ecs_albs" {
 
 ## Resources
 
+resource "aws_cloudwatch_log_group" "task_logs" {
+  name = "${var.stack_name}"
+  retention_in_days = 7
+  tags = {
+    Environment = "${var.stack_name}"
```

Comment:
These tags could be more consistent with the rest of our tags in the other projects please (e.g. adding `Stackname`, using additional vars as well)

Reply:
can you be more specific? I can't find any other examples of us setting Environment in this repo

Reply:
https://github.com/alphagov/prometheus-aws-configuration-beta/blob/master/stacks/staging.tfvars Is where we set the environment as an additional tag and then we set all the tags on resources like https://github.com/alphagov/prometheus-aws-configuration-beta/blob/master/terraform/projects/infra-security-groups/main.tf#L85